src/renderer/gl_engine/tvgGlRenderer.cpp

```diff
@@ -69,6 +69,11 @@ bool GlRenderer::target(int32_t id, uint32_t w, uint32_t h)
 
     mTargetFboId = static_cast<GLint>(id);
 
+    //TODO: It's not allow to draw onto the main surface. Need to confirm the policy.
+    if (mTargetFboId == 0) {
+        GL_CHECK(glGetIntegerv(GL_FRAMEBUFFER_BINDING, &mTargetFboId));
+    }
```

Comment:
If with EFL, the `mTargetFboId` is not zero, does it mean that at https://github.com/thorvg/thorvg/blob/main/examples/Path.cpp#L113 this point in time, the framebuffer corresponding to GLView has been initialized?

Reply:
@RuiwenTang Yes. Actually glview surface fbo in EFL has been initialized at the point, However the gl binding api does not return it. (that's the problem)

Reply:
But the comment says that drawing to the main surface (fbo 0) is not allowed. I think this description is not accurate.

Reply:
Because currently it's aborted, not allowed to draw on the main surface. It's a todo fix. please fix it. :-)